### ISSUE TITLE
VEYON-MASTER STOP WORKING.

### ISSUE THREAD

**Issue by USMPFMH** [veyon/veyon#304]
Hello everyone, I have a problem every time I start the veyon master it closes automatically, appearing a window indicating VEYON-MASTER STOP WORKING.

I attach the screen print of the error. Please I need your help

[error veyon.docx](https://github.com/veyon/veyon/files/1868813/error.veyon.docx)

**Comment by tobydox** [comment]
What operating system are you running? Can you open Veyon Configurator? Which changes did you make to the configuration?

**Comment by USMPFMH** [comment]
HI i use WINDOWS 7 PRO 64 BITS. I attach a file with a print screen of veyon configuration. Thank so much. 
[Veyon Configurator.docx](https://github.com/veyon/veyon/files/1872117/Veyon.Configurator.docx)

**Comment by jamj2000** [comment]
Esencially it may be 3 causes:

1. bug in veyon-master. 
2. misfunctionality in your Windows in master. 
3. trouble in lan.

In the first case you can try in diferent circunstances and point in which one it fails.

In the second case, I guess you could:

- Install veyon-master in another computer, preferebly on fresh install if you can and/or diferent version.
- Add less computers to list.

In the third case, check switch and net traffic, and ...

**Comment by USMPFMH** [comment]
yes, i´ll try again with the upgrade but in a new computer and the automatic adjustment of computer screen sizes is disable. I think the problem is the computer, i have seen in the event viewer in windows and it showed me a message indicating that there were errors in some dll files. I'm going to take a print screen and post it so you can see it. I will notify about the new version of Veyon 4.0.6. Thanks again.

**Comment by USMPFMH** [comment]
a question, when installing the new version is there any option to import the previous registration key in the veyon configurator? because I have 150 client computers and would have to configure a new registry key from each computer and that would take a lot of time.

**Comment by tobydox** [comment]
You don't have to export and import all settings on a simple upgrade from 4.0.x to 4.0.7. However you can automate this though the CLI tool Veyon Control (veyon-ctl) - please refer to our admin manual for details ;-) Please respond and re-open the issue if it still persists.

**Comment by USMPFMH** [comment]
hi,
I installed veyon 4.0.6 on a new computer  with win 7 pro 64 bit , it has all the updates installed. I have started the VEYON MASTER and again I have the same problem, I have a message that tells me that in the TEMP folder there are those 3 attached files that detail the error. What I notice in one of the files is that there is a problem with libqca-ossl.dll.
[ERROR.zip](https://github.com/veyon/veyon/files/2032588/ERROR.zip)

i need to resolve this as soon as possible. Thanks.
